src/DotVVM.Framework/Hosting/ErrorPages/ErrorPageTemplate.cs

```diff
@@ -1,705 +1,324 @@
-}
-            
-            #line default
-            #line hidden
-            this.Write("\t\t<hr />\r\n\t\t");
-            
-            #line 89 "C:\dev\dotvvm\src\DotVVM.Framework\Hosting\ErrorPages\ErrorPageTemplate.tt"
-foreach(var formatter in Formatters){
-            
-            #line default
-            #line hidden
-            this.Write("\t\t<div class=\"container\" id=\"container_");
-            
-            #line 90 "C:\dev\dotvvm\src\DotVVM.Framework\Hosting\ErrorPages\ErrorPageTemplate.tt"
-            this.Write(this.ToStringHelper.ToStringWithCulture(formatter.Id));
-            
-            #line default
-            #line hidden
-            this.Write("\">\r\n\t\t\t");
-            
-            #line 91 "C:\dev\dotvvm\src\DotVVM.Framework\Hosting\ErrorPages\ErrorPageTemplate.tt"
-formatter.WriteBody(this);
-            
-            #line default
-            #line hidden
-            this.Write("\t\t</div>\r\n\t\t");
-            
-            #line 93 "C:\dev\dotvvm\src\DotVVM.Framework\Hosting\ErrorPages\ErrorPageTemplate.tt"
-}
-            
-            #line default
-            #line hidden
-            this.Write("\r\n\t\t</div>\r\n\r\n\t\t<p>&nbsp;</p>\r\n\r\n\t</body>\r\n</html>\r\n\r\n\r\n\r\n\r\n");
-            return this.GenerationEnvironment.ToString();
-        }
-        
-        #line 105 "C:\dev\dotvvm\src\DotVVM.Framework\Hosting\ErrorPages\ErrorPageTemplate.tt"
+        private readonly StringBuilder builder = new StringBuilder();
 
-    public int ErrorCode { get; set; }
+        public const string InternalCssResourceName = "DotVVM.Framework.Resources.Styles.DotVVM.Internal.css";
 
-    public string ErrorDescription { get; set; }
+        public ErrorPageTemplate(
+            int errorCode,
+            string errorDescription,
+            string summary,
+            IErrorSectionFormatter[] formatters)
+        {
+            ErrorCode = errorCode;
+            ErrorDescription = errorDescription;
+            Summary = summary;
+            Formatters = formatters;
+        }
 
-    public string Summary { get; set; }
+        public int ErrorCode { get; }
+        public string ErrorDescription { get; }
+        public string Summary { get; }
+        public IErrorSectionFormatter[] Formatters { get; }
 
-    public IErrorSectionFormatter[] Formatters { get; set; }
+        public void WriteText(string str)
+        {
+            Write(WebUtility.HtmlEncode(str));
+        }
 
-    public void WriteText(string str)
-    {
-        Write(WebUtility.HtmlEncode(str));
-    }
+        public void WriteUnencoded(string str)
+        {
+            Write(str);
+        }
+        
+        public string TransformText()
+        {
+            // head
+            Write(
+$@"<!DOCTYPE html>
+<html>
+    <head>
+        <title>Server Error in Application</title>
+        <meta charset=""UTF-8"" />
```

Comment:
```suggestion
        <meta charset=UTF-8 />
```

You can easily save yourself this annoying quoting quotes